examples/with-nextjs-13/app.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@web3-onboard/tallyho'
```

Comment:
@taylorjdawson Out of curiosity, what does this do and can we fix it at the package level?

Reply:
Actually I think the error may be gone now I am no longer seeing it 🤔